README.yaml

```diff
@@ -0,0 +1,247 @@
+  - name: "terraform-datadog-monitor"
+    description: "Terraform module to configure and provision Datadog monitors from a YAML configuration, complete with automated tests."
+    url: "https://github.com/cloudposse/terraform-datadog-monitor"
+  - name: "terraform-opsgenie-incident-management"
+    description: "Terraform module to provision Opsgenie resources from YAML configurations using the Opsgenie provider, complete with automated tests."
+    url: "https://github.com/cloudposse/terraform-opsgenie-incident-management"
+  - name: "terraform-aws-components"
+    description: "Catalog of reusable Terraform components and blueprints for provisioning reference architectures."
+    url: "https://github.com/cloudposse/terraform-aws-components"
+  - name: "reference-architectures"
+    description: "Get up and running quickly with one of our reference architecture using our fully automated cold-start process."
+    url: "https://github.com/cloudposse/reference-architectures"
+
+# List any resources helpful for someone to get started. For example, link to the hashicorp documentation or AWS documentation.
+references:
+  - name: "Terraform Standard Module Structure"
+    description: "HashiCorp's standard module structure is a file and directory layout we recommend for reusable modules distributed in separate repositories."
+    url: "https://www.terraform.io/docs/modules/index.html#standard-module-structure"
+  - name: "Terraform Module Requirements"
+    description: "HashiCorp's guidance on all the requirements for publishing a module. Meeting the requirements for publishing a module is extremely easy."
+    url: "https://www.terraform.io/docs/registry/modules/publish.html#requirements"
+  - name: "Terraform Version Pinning"
+    description: "The required_version setting can be used to constrain which versions of the Terraform CLI can be used with your configuration."
+    url: "https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version"
+  - name: "Terraform `templatefile` Function"
+    description: "`templatefile` reads the file at the given path and renders its content as a template using a supplied set of template variables."
+    url: "https://www.terraform.io/docs/configuration/functions/templatefile.html"
+  - name: "Terraform `template_file` data source"
+    description: "The `template_file` data source renders a template from a template string, which is usually loaded from an external file."
+    url: "https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file"
+  - name: "Deepmerge"
+    description: "Terraform module to perform a deep map merge of standard Terraform maps and objects."
+    url: "https://github.com/Imperative-Systems-Inc/terraform-modules/tree/master/deepmerge"
+
+# Short description of this project
+description: |-
+  Terraform module to convert local and remote YAML configuration templates into deep-merged maps
```

Comment:
This module is more about dealing with stacks. These are implementation details.